src/main/java/net/greatstart/dao/hibernate/RoleDaoImpl.java

```diff
@@ -0,0 +1,53 @@
+package net.greatstart.dao.hibernate;
+
+import net.greatstart.dao.RoleDao;
+import net.greatstart.model.Role;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class RoleDaoImpl implements RoleDao {
+    private SessionFactory sessionFactory;
+
+    @Autowired
+    public RoleDaoImpl(SessionFactory sessionFactory) {
+        this.sessionFactory = sessionFactory;
+    }
+
+    @Override
+    public boolean create(Role item) {
+        return false;
+    }
+
+    @Override
+    public boolean update(Role item) {
+        return false;
+    }
+
+    @Override
+    public boolean delete(Role item) {
+        return false;
+    }
+
+    @Override
+    public Role getById(long id) {
+        return sessionFactory.getCurrentSession()
+                .get(Role.class, id);
+    }
+
+    @Override
+    public List<Role> getAll() {
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)